src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a person's block in their address.
+ * 
+ * @author Clarence
+ */
+public class Block {
+    private final int block;
+    
+    private static final String BLOCK_VALIDATION_REGEX = "\\d+"; // any integer
+    
+    public Block(int block) {
+        super();
```

Comment:
why are you calling a super class constructor here and what is the superclass here? Not necessary.